ci/build-deb

```diff
@@ -17,16 +17,21 @@ if ! command -V cargo-deb > /dev/null 2>&1; then
     exit 1
 fi
 
+if ! command -V asciidoctor > /dev/null 2>&1; then
+    echo "asciidoctor command missing" >&2
+    exit 1
+fi
```

Comment:
This LGTM. The `a2x` support in the build script is legacy I think, so it's likely to be removed.